### PR TITLE
day 18-q2 py, day21-q2, day 22-q1, 23-q1, 19-q3, 21-q3

### PR DESCRIPTION
**Pull request by Tech-neophyte**

### CODE REVIEW

**Comment by Tech-neophyte** [comment]
#421 #436

**Comment by Tech-neophyte** [comment]
#438

**Comment by Tech-neophyte** [comment]
#446 added

**Comment by Tech-neophyte** [comment]
#418

**Comment by Tech-neophyte** [comment]
python3 and cpp code added. #445

File: Day-18/q2: Search in Rotated Sorted Array/Question.md
Comment (bh-g) [inline Day-18/q2: Search in Rotated Sorted Array/Question.md:1]: question is already added, you can delete this file.

File: Day-21/Q3/Tech--neophyte--pc.md
Comment (bh-g) [inline Day-21/Q3/Tech--neophyte--pc.md:1]: You can move this file to Day-21/q3: Unique Paths

File: day-23/q1/Tech-neophyte--c.md
Comment (bh-g) [inline day-23/q1/Tech-neophyte--c.md:1]: pls move this file to Day-23/q1: Minimum falling path sum

**Review by bh-g — CHANGES_REQUESTED** [review]
Please make these changes and I can merge this PR :)

File: Day-18/q2: Search in Rotated Sorted Array/Question.md
Comment (Tech-neophyte) [reply Day-18/q2: Search in Rotated Sorted Array/Question.md:1]: Deleted :)

File: Day-21/Q3/Tech--neophyte--pc.md
Comment (Tech-neophyte) [reply Day-21/Q3/Tech--neophyte--pc.md:1]: @bh-g done :)

File: day-23/q1/Tech-neophyte--c.md
Comment (Tech-neophyte) [reply day-23/q1/Tech-neophyte--c.md:1]: Hey! the solutions were already added for this one so, I deleted the file. Rest of the changes are done.. @bh-g

**Comment by Tech-neophyte** [comment]
> Please make these changes and I can merge this PR :)

Done with the changes :)

**Pull request merged**